<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1231</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tableau Véhicule</string>
     </attribute>
     <widget class="QCheckBox" name="filtreKilometrageCheckBox">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>320</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>kilometrage</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="vehiculesTableView">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>571</width>
        <height>341</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreImmatriculationCheckBox">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>200</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Immatriculation</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreDescriptionCheckBox">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>240</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>description</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreLongitudeCheckBox">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>400</y>
        <width>77</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>longitude</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreLatitudeCheckBox">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>360</y>
        <width>77</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>latitude</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreDisponibleCheckBox">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>440</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>disponibilité</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreCouleurCheckBox">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>280</y>
        <width>77</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>couleur</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>140</y>
        <width>121</width>
        <height>341</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>133</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Filtres :</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>lineEdit_2</zorder>
     <zorder>filtreKilometrageCheckBox</zorder>
     <zorder>vehiculesTableView</zorder>
     <zorder>filtreImmatriculationCheckBox</zorder>
     <zorder>filtreDescriptionCheckBox</zorder>
     <zorder>filtreLongitudeCheckBox</zorder>
     <zorder>filtreLatitudeCheckBox</zorder>
     <zorder>filtreDisponibleCheckBox</zorder>
     <zorder>filtreCouleurCheckBox</zorder>
     <zorder>lineEdit_3</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tableau Conducteur</string>
     </attribute>
     <widget class="QTableView" name="conducteursTableView">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>701</width>
        <height>451</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtrePrenomConducteurCheckBox">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>110</y>
        <width>77</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreNomConducteurCheckBox">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>140</y>
        <width>77</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreEmailConducteurCheckBox">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>170</y>
        <width>77</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>60</y>
        <width>113</width>
        <height>141</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>60</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Filtres :</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>lineEdit_4</zorder>
     <zorder>conducteursTableView</zorder>
     <zorder>filtrePrenomConducteurCheckBox</zorder>
     <zorder>filtreNomConducteurCheckBox</zorder>
     <zorder>filtreEmailConducteurCheckBox</zorder>
     <zorder>lineEdit_5</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Tableau Client</string>
     </attribute>
     <widget class="QTableView" name="clientsTableView">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>701</width>
        <height>441</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreNomClientCheckBox">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>140</y>
        <width>77</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreAdresseCheckBox">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>170</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse Chantier</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>90</y>
        <width>131</width>
        <height>111</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>90</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Filtres :</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>lineEdit_6</zorder>
     <zorder>clientsTableView</zorder>
     <zorder>filtreNomClientCheckBox</zorder>
     <zorder>filtreAdresseCheckBox</zorder>
     <zorder>lineEdit_7</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tableau Mission</string>
     </attribute>
     <widget class="QTableView" name="missionsTableView">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>811</width>
        <height>481</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="editMissionButton">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>110</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier Mission</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreLavageCheckBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>77</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Lavage</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtrePleinCheckBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>77</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Plein</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>161</width>
        <height>281</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Filtres :</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreDeltaDistanceCheckBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Delta Distance</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreImmatriculationMissionCheckBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Immatriculation</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreHorodatageDepartCheckBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Horodatage Départ</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreHorodatageArriveeCheckBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Horodatage Arrivée</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="remboursementLavageCheckBox">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>40</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Remboursement Lavage</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreRemboursementLavageCheckBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Remboursement Lavage</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filtreRemboursementPleinCheckBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Remboursement PLein</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="remboursementPleinCheckBox">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>70</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Remboursement Plein</string>
      </property>
     </widget>
     <zorder>lineEdit_8</zorder>
     <zorder>missionsTableView</zorder>
     <zorder>editMissionButton</zorder>
     <zorder>filtreLavageCheckBox</zorder>
     <zorder>filtrePleinCheckBox</zorder>
     <zorder>lineEdit_9</zorder>
     <zorder>filtreDeltaDistanceCheckBox</zorder>
     <zorder>filtreImmatriculationMissionCheckBox</zorder>
     <zorder>filtreHorodatageDepartCheckBox</zorder>
     <zorder>filtreHorodatageArriveeCheckBox</zorder>
     <zorder>remboursementLavageCheckBox</zorder>
     <zorder>filtreRemboursementLavageCheckBox</zorder>
     <zorder>filtreRemboursementPleinCheckBox</zorder>
     <zorder>remboursementPleinCheckBox</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1310</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
